conda-forge.yml

```diff
@@ -1,6 +1,7 @@
 travis:
   secure:
     BINSTAR_TOKEN: I+L4bYUiz09ojMmKlRth2u9kOsScZLGdVLE5pGRLh5mesyuztpTQ44WiDJbIj5VnBLv0/9xvUekIJz0Ok1dQ+QAVHKrUizPAglIQLWG1jX/eytbyK+RZCYIdqbOSxRrdkLuaLyGmP8ARUw8xywSRCzRGGDFfs31dPtsrXGDSYxrqkzVU8EiehkC93aupIyoJWTGRAQ4PA88AhdWTGdfBkIfmx0sNuBbH1FzKi3qpXc21ThjnT6O7JU7Nezq2yEW5GO1WD0sXgY414QALyKHl/Gf7mLO1fOd4EcLHxdDTC+BiQmCBJZpK8u8JYO12zTV0QrI6AAvvY0pE4bpbdHFlmO9oQ/NnVE9qCPLybh6FC0AH8nnTBlzDVna1shWjzf2zvvATZSrLQ3A4Aez61B5//Ps7c33U2/O3IrDl/VXW5VXPagNfrI5zHTG7I6zt20BRNmQmPwlT6Lt7h+wCorlR21sqqf1zUaMdP9xPEhT0JwreuY/jz9hzok7S0lGzMzhaMgD6F7o5QXtsRp9LQ26FUAVG4HVSQs4SABiC9LAoDyIFj3+BZurHQ1FFaBz58AGVgnzkkRONggAqlCMGi1O5lMB5vu86uTOnKGpXBcBcwMHplCWRN74MokhSZ2CYlaWqQn405xy/HPNfu97e9OFZObuJo4l1HQpXd6GjFE8wIqM=
+compiler_stack: comp7
```

Comment:
I'm not sure this is something you should be editing. Is this not autogenerated by conda smithy?

Reply:
I am wrong about that. I see this is an option: https://conda-forge.org/docs/maintainer/conda_forge_yml.html

Reply:
I did not update this file; `conda-smithy` is responsible.